src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1432,7 +1470,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 LookupInInterfaces(lookupResult, container, lookIn, processed, name, arity, options, binder, methodsOnly, useSiteDiagnostics)
 
                 ' If no viable or ambiguous results, look in Object.
-                If Not lookupResult.IsGoodOrAmbiguous AndAlso (options And LookupOptions.NoSystemObjectLookupForInterfaces) = 0 Then
+                If lookupResult?.IsGoodOrAmbiguous <> True AndAlso (options And LookupOptions.NoSystemObjectLookupForInterfaces) = 0 Then
```

Comment:
>lookupResult?.IsGoodOrAmbiguous <> True [](start = 19, length = 39)

This still looks suspicious. In VB ```lookupResult?.IsGoodOrAmbiguous <> True``` will be ```Boolean?```  and null value is not true, but I think we would want to continue when ```lookupResult``` is Nothing.

Reply:
> The new expression is intended to treat Nothing the same as empty, and also produce True for the expression.

Why do you think so? Comparing nullable null to anything yields null in VB, not true, not false 

---
In reply to: [184547523](https://github.com/dotnet/roslyn/pull/26398#discussion_r184547523) [](ancestors = 184547523)

Reply:
Is there any short form here? The alternative I know of is this expression:

```
lookupResult Is Nothing OrElse Not lookupResult.IsGoodOrAmbiguous
```

Reply:
> Is there any short form here

I think this is fine. There is an alternative ```Not If(lookupResult?.IsGoodOrAmbiguous, False)```, but I think explicit null check is easier to follow.

---
In reply to: [184820315](https://github.com/dotnet/roslyn/pull/26398#discussion_r184820315) [](ancestors = 184820315)